submissions/IRETIOGEDENGBE/css/body.css

```diff
@@ -0,0 +1,78 @@
+body{
```

Comment:
It is not exactly clear what page uses this stylesheet. Can you rename it after the page it styles?